plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -65,7 +65,7 @@ const PodContainerConfigSection = ({containerConfig, appConfig, onEditClick, ind
       </ConfigurationMapRow>
 
       {/* Resources */}
-      {fields.resources.cpus && (
+      {!!fields.resources.cpus && (
```

Comment:
I prefer the use of `Boolean` over the bang bang. And would echo @mlunoe's comment and add, that, we should try to be mindful about new patterns and add them to our coding styles first rather than add them to the codebase and then to the code styles.